### PR TITLE
Features Release - Sorting Scheldules Intervals

### PR DESCRIPTION
**Pull request by butuzov**
Added ability to sort schedules intervals.
---
After long way of code formatting and spelling fixes.

### CODE REVIEW

**Comment by jrfnl** [comment]
Looking good, just ran it through WPCS one last time and one more (ok, two more) small fixes & we're done...

File: class-debug-bar-cron.php [hunk 1]
```diff
@@ -380,6 +382,21 @@ private function display_schedules() {
...
+		/**
+		 * Sorting method for cron scheldules. Order by schedules interval.
+		 *
+		 * @param array $a First element of comparison pair.
+		 * @param array $b Second element of comparison pair.
+		 *
+		 * @return int Return 1 if $a argument 'interval' greater then $b argument 'interval', 0 if both intervals equivalent and -1 otherwise.
+		 */
+		function schedules_sorting( $a, $b ){
+			if ( (int) $a['interval'] === (int) $b['interval'] ){
```
Comment (jrfnl) [inline class-debug-bar-cron.php]: WP Coding standards demands a space between these `){` => `) {`. Line 393 and 394.

File: class-debug-bar-cron.php [hunk 2]
```diff
@@ -380,6 +382,21 @@ private function display_schedules() {
...
+				return 0;
+			} else {
+				return (int) $a['interval'] > (int) $b['interval'] ? 1 : -1;
```
Comment (jrfnl) [inline class-debug-bar-cron.php]: Ternary comparisons should be bracketed to ensure that there can be no confusion as to what is part of the ternary and what should be returned.

So this should be changed to:
```php
return ( ( (int) $a['interval'] > (int) $b['interval'] ) ? 1 : -1 );
```

File: class-debug-bar-cron.php [hunk 3]
```diff
@@ -380,6 +382,21 @@ private function display_schedules() {
...
 
+		/**
+		 * Sorting method for cron scheldules. Order by schedules interval.
+		 *
+		 * @param array $a First element of comparison pair.
+		 * @param array $b Second element of comparison pair.
+		 *
+		 * @return int Return 1 if $a argument 'interval' greater then $b argument 'interval', 0 if both intervals equivalent and -1 otherwise.
+		 */
+		function schedules_sorting( $a, $b ){
```
Comment (jrfnl) [inline class-debug-bar-cron.php]: Here as well please... :+1: `){` => `) {`

**Comment by butuzov** [comment]
Corrected. I definitely need to spend weekend to improve Git AND WP CodeStandards.

**Comment by jrfnl** [comment]
Awesome!!! Thank you for your patience and willingness to work on this with me.

If there is anything I can do to help you with getting to grips with GIT and the WP Coding Standards, please drop me a line & I'll try to help where I can.

**Comment by butuzov** [comment]
No need to worry. I will try to refresh my knowledges with codecademy course, as for wpcs I just need to read and force myself to folow it. As I said before "a bit shame..." for wp developer with 8+y experience...

I also will try to finish all discussed fixes and corrections for other PR till next Monday. Thank you for your help and patience

**Comment by jrfnl** [comment]
Sounds good !

If you haven't already, you may also want to install PHPCS and the WPCS sniffs and play around with that a bit.
https://github.com/squizlabs/PHP_CodeSniffer
https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/

**Comment by butuzov** [comment]
Intresting! Thank you!

**Pull request merged**